### ISSUE TITLE
No way to select a bookmark as destination in "From/To" route screen

### ISSUE THREAD

**Issue by amykyta3** [mapsme/omim#5121]
Version 7.0.5-Google
On Android 6.0.1

1. When in map view, press the "Route" menu button
2. Tap the `To` field
3. In destination search prompt:
Search only shows locations on map. There is no way to select one of my bookmarks from this screen. Of course, the workaround is to navigate to the bookmark itself, and press `Route To`, but it would be intuitive to be able to do it from this screen as well. Solution could be for bookmarks to also show up in search, and/or a separate tab to select from a list of all bookmarks, sorted by distance.

**Comment by burivuh** [comment]
Hi! We're keeping this feature in mind, thank you!

**Comment by Norbert80** [comment]
I vote for this one!

**Comment by MagTun** [comment]
This is a really important feature, could you please implement it.

**Comment by WiliTest** [comment]
I agree, in google map, i just open the "find" tool, and search "Home" and to "john" and it's done (the name of a bookmark are simply linked to an adress). 

I mostly use google map simply because it offers this basic feature.

**Comment by burivuh** [comment]
Hi!

Can you, please, tell us in details, what is needed?

With the current interface you have a Bookmarks floating button (upon a Search button on the left) on the Route Planning screen. You can select you bookmark there and click on Route to or Route from.

**Comment by MagTun** [comment]
In details:
Let's say I am pressing somewhere on the map and a blue circle appear with the menu "route from". Then I select the menu "add a destination to plan a route". From this page I would like to be able to select a bookmark., but it's not possible.

I know I could do a route the other way around: first find my bookmark then create a "route to" then select the location with a blue circle BUT sometimes these 2 locations are very far appart and thus this option is not a convenient.
